bcda/web/router.go

```diff
@@ -23,14 +25,9 @@ func NewAPIRouter() http.Handler {
 	}
 	FileServer(r, "/api/v1/swagger", http.Dir(swagger_path))
 
-	// Don't even load up the route in production
 	if os.Getenv("DEPLOYMENT_TARGET") != "prod" {
-		// Serve up the static site
-		jekyll_path := "./_site"
-		if _, err := os.Stat(jekyll_path); os.IsNotExist(err) {
-			jekyll_path = "../_site"
-		}
-		FileServer(r, "/", http.Dir(jekyll_path))
+		r.Get("/", userGuideRedirect)
+		r.Get(`/{p:(user_guide|encryption|decryption_walkthrough).html}`, userGuideRedirect)
```

Comment:
Not sure if this is the final implementation of the user guide, but currently:
The sandbox user guide is here: https://bcda.cms.gov/sandbox/user-guide/index.html
And the production user guide is here: https://bcda.cms.gov/user_guide.html

Reply:
We've done pretty URLs as of a few minutes ago. Sbx user guide is /sandbox/user-guide/ , prod user guide is production/user-guide/ .

Reply:
The old user guide had 4 pages: index, user guide, encryption, decryption.

index -> index
sandbox.bcda.cms.gov/user_guide.html -> bcda.cms.gov/sandbox/user-guide/
encryption AND decryption pages from sandbox.bcda.cms.gov should both redirect to bcda.cms.gov/decryption/ .

Reply:
OR we could follow Ryan's suggestion and just send any of the old sandbox pages to the new index page. Also totally great.

Reply:
That is much simpler.  I like that idea!